pyproject.toml

```diff
@@ -25,15 +25,15 @@ kgcl-schema = "^0.3.1"
 funowl = "^0.1.12"
 gilda = {version = "^0.10.1", optional = true}
 kgcl-rdflib = "^0.3.0"
-pystow = "^0.5.0"
+pystow = "^0.4.4"
```

Comment:
ideally we would not do this

Reply:
Reverted in 9e38b39c9106440256fe1ad5407bd3c4187d0338